gbus/worker.go

```diff
@@ -269,9 +289,14 @@ func (worker *worker) processMessage(delivery amqp.Delivery, isRPCreply bool) {
 				log.Fields{"message-name": bm.PayloadFQN,
 					"message-type": bm.Semantics}).
 			Warn("Message received but no handlers found")
+		span.LogFields(slog.String("grabbit", "no handlers found"))
 		// worker.log("Message received but no handlers found\nMessage name:%v\nMessage Type:%v\nRejecting message", bm.PayloadFQN, bm.Semantics)
 		//remove the message by acking it and not rejecting it so it will not be routed to a deadletter queue
-		worker.Ack(delivery)
+		err = worker.Ack(delivery)
```

Comment:
It's probably worth invoking it with SafeWithRetries

Reply:
worker.Reject already implements safe retries https://github.com/wework/grabbit/blob/fix-opentracing/gbus/worker.go#L192